account_asset_analytic/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# flake8: noqa
```

Comment:
Yeah, but you should use the same QA tools that in Travis (why are you not using https://github.com/OCA/maintainer-quality-tools/tree/master/git?), and not make this modifications. With this, you are disabling the check in all models files.